compiler/luci/service/src/CircleShapeInference.cpp

```diff
@@ -28,23 +28,6 @@
 namespace
 {
 
-std::ostream &operator<<(std::ostream &os, const loco::TensorShape &tensor_shape)
```

Comment:
why is this removed ?

Reply:
I moved it to `compiler/luci/service/src/CircleShapeInferenceHelper.cpp` because I thought it would be more appropriate place for this.
If you don't think so, I'll rollback this change.